omod/src/main/java/org/openmrs/module/owa/web/controller/OwaRestController.java

```diff
@@ -162,12 +164,15 @@
 				HttpSession session = request.getSession();
 				if (!url.isEmpty()) {
 					InputStream inputStream = ModuleUtil.getURLStream(downloadUrl);
-					log.warn("url pathname: " + downloadUrl.getPath());
-					String fileName = downloadUrl.getQuery().substring(downloadUrl.getQuery().lastIndexOf("=") + 1);
+					log.warn("Url pathname: " + downloadUrl.getPath());
```

Comment:
Not sure this is the correct use of `log.warn`. Consider using `log.info`

Reply:
Have changed to using log.info in the necessary locations